tests/integration/lineage.js

```diff
@@ -0,0 +1,489 @@
+const { expect } = require('chai');
+const sinon = require('sinon').sandbox.create();
+const lineageFactory = require('../../shared-libs/lineage/src/lineage');
+const { db } = require('../utils');
+
+const circular_areaId = 'circular_area';
+const circular_chw = {
```

Comment:
I think for me the tests would be more readable if either
a) these `const`s were defined with upper-case identifieds (e.g. const `CIRCULAR_CHW`), or
b) these were all defined together in a map, so that code would read e.g. `FIXTURES.circular_chw`
Probably not worth changing if you're happy with the current notation, though.

Reply:
No I find it quite frustrating constantly moving back to the top of the file and not really knowing what's going on when I'm within the test, I might try and find a better way to do all of this